zizroc/villager
# Fix Build

This PR fixes the build with respect to namespacing functions.

## To Test:

1. Checkout
2. Click 'Check' in RStudio
3. Click 'Install and Restart'
4. Make sure there weren't any errors
